test/test_repolib.py

```diff
@@ -33,10 +34,13 @@
 
 from .stubs import StubProductCertificate, \
         StubProduct, StubEntitlementCertificate, StubContent, \
-        StubProductDirectory, StubConsumerIdentity, StubEntitlementDirectory
+        StubProductDirectory, StubConsumerIdentity, StubEntitlementDirectory, \
+        StubDeb822
 from subscription_manager.repolib import RepoActionInvoker, \
         RepoUpdateActionCommand, YumReleaseverSource, YumPluginManager
-from subscription_manager.repofile import Repo, TidyWriter, YumRepoFile
+# Stub import of Deb822 which is used in AptRepoFile
+sys.modules['debian.deb822.Deb822'] = StubDeb822
```

Comment:
I will have to trigger jenkins again for this issue... I think that this change could work.

Reply:
Did you triger the jenkins build again @jirihnidek  ?